Library/Formula/ploticus242.rb

```diff
@@ -0,0 +1,18 @@
+class Ploticus242 < Formula
```

Comment:
This should probably be plain `Ploticus` and live in file `ploticus.rb`. Homebrew doesn't bake version numbers in to formula names.